packages/core/modules-sdk/src/__mocks__/stock-location-module.ts

```diff
@@ -6,7 +6,7 @@ export const StockLocationModule = {
     label: "StockLocationService",
     isRequired: false,
     isQueryable: true,
-    dependencies: ["eventBusService"],
+    dependencies: ["eventBusModuleService"],
```

Comment:
nit: ModuleRegistrationNames

Reply:
@adrien2p I think these mocks will get removed anyway so it didn't matter